diff -urN usb-modeswitch-1.1.3/Makefile usb-modeswitch-1.1.3.new//Makefile
--- usb-modeswitch-1.1.3/Makefile	2010-06-21 21:27:29.000000000 +0200
+++ usb-modeswitch-1.1.3.new//Makefile	2010-07-23 16:31:27.765913994 +0200
@@ -1,7 +1,10 @@
 PROG        = usb_modeswitch
 VERS        = 1.1.3
 CC          = gcc
-CFLAGS      += -Wall -l usb
+CFLAGS      += -Wall
+CPPFLAGS    = $(NULL)
+LDFLAGS     = $(NULL)
+LIBS        = -lusb
 RM          = /bin/rm -f
 OBJS        = usb_modeswitch.c
 PREFIX      = $(DESTDIR)/usr
@@ -15,7 +18,7 @@
 all:        $(PROG)
 
 $(PROG): $(OBJS)
-	$(CC) -o $(PROG) $(OBJS) $(CFLAGS) $(LDFLAGS)
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o $(PROG) $(OBJS) $(LIBS)
 
 clean:
 	$(RM) usb_modeswitch
