diff -urNp -x '*.orig' usb-modeswitch-2.6.0.org/Makefile usb-modeswitch-2.6.0/Makefile
--- usb-modeswitch-2.6.0.org/Makefile	2019-11-11 21:45:13.000000000 +0100
+++ usb-modeswitch-2.6.0/Makefile	2022-02-15 16:31:14.356333430 +0100
@@ -2,6 +2,8 @@ PROG        = usb_modeswitch
 VERS        = 2.6.0
 CC          ?= gcc
 CFLAGS      += -Wall -Wno-deprecated-declarations
+CPPFLAGS    =
+LDFLAGS     =
 LIBS        = `pkg-config --libs --cflags libusb-1.0`
 RM          = /bin/rm -f
 OBJS        = usb_modeswitch.c
@@ -26,7 +28,7 @@ all-with-dynlink-dispatcher: $(PROG) dis
 all-with-statlink-dispatcher: $(PROG) dispatcher-statlink
 
 $(PROG): $(OBJS) usb_modeswitch.h
-	$(CC) -o $(PROG) $(OBJS) $(CFLAGS) $(LIBS) $(LDFLAGS)
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o $(PROG) $(OBJS) $(LIBS)
 
 dispatcher-script: usb_modeswitch_dispatcher.tcl
 	DISPATCH=dispatcher-script
