--- usb-modeswitch-2.0.1/Makefile.orig	2013-09-03 19:22:05.000000000 +0200
+++ usb-modeswitch-2.0.1/Makefile	2013-09-14 07:06:41.048926415 +0200
@@ -2,6 +2,8 @@
 VERS        = 2.0.1
 CC          ?= gcc
 CFLAGS      += -Wall
+CPPFLAGS    =
+LDFLAGS     =
 LIBS        = `pkg-config --libs --cflags libusb-1.0`
 RM          = /bin/rm -f
 OBJS        = usb_modeswitch.c
@@ -36,7 +38,7 @@
 static: $(PROG) dispatcher-static
 
 $(PROG): $(OBJS) usb_modeswitch.h
-	$(CC) -o $(PROG) $(OBJS) $(CFLAGS) $(LIBS) $(LDFLAGS)
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o $(PROG) $(OBJS) $(LIBS)
 
 jim/libjim.so:
 	cd jim && CFLAGS="$(CFLAGS)" CC="$(CC)" ./configure $(JIM_CONFIGURE_OPTS) --shared
