--- usb-modeswitch-1.1.7/Makefile.makefile	2011-02-23 00:07:38.000000000 +0100
+++ usb-modeswitch-1.1.7/Makefile	2011-04-28 12:08:16.000000000 +0200
@@ -2,6 +2,8 @@ PROG        = usb_modeswitch
 VERS        = 1.1.7
 CC          = gcc
 CFLAGS      += -Wall
+CPPFLAGS    = $(NULL)
+LDFLAGS     = $(NULL)
 LIBS        = -l usb
 RM          = /bin/rm -f
 OBJS        = usb_modeswitch.c
@@ -16,7 +18,7 @@ MANDIR      = $(PREFIX)/share/man/man1
 all:        $(PROG)
 
 $(PROG): $(OBJS)
-	$(CC) -o $(PROG) $(OBJS) $(CFLAGS) $(LIBS) $(LDFLAGS)
+	$(CC) $(CPPFLAGS) $(CFLAGS) $(LDFLAGS) -o $(PROG) $(OBJS) $(LIBS)
 
 clean:
 	$(RM) usb_modeswitch
